src/core/xfa/bind.js

```diff
@@ -643,6 +643,9 @@ class Binder {
                 : dataNode[$namespaceId];
             match = child[$data] = new XmlObject(nsId, child.name);
             if (this.emptyMerge) {
+              if (child.name === "nomin" && child.occur.min === 0) {
```

Comment:
Could you point out where in the specifications you found that an element called `nomin` is useless ?
Afaik, an element with a child `<occur min="0" ...>` is useless when no data can be bound in this element.

Reply:
Have a look on:
https://github.com/mozilla/pdf.js/blob/6a15973a1b26288f981d971fab1f54bb4c98d8be/src/core/xfa/bind.js#L666
The node is useless when `match` is null (nothing has been found in the data tree which matches with the child from the template tree) and `min <= 0` (i.e. `min===0`).
But because of a bug, with the pdf in the issue, we never hit this path.
So there are several possibilities:
 - we've a `match` when we shouldn't;
 - we've `min > 0` when we shouldn't;
 - we never go there because there is a `continue` or a `return` somewhere.